src/styles/main.scss

```
@@ -1,3 +1,61 @@
-body {
-  margin: 0;
+@import "./variables";
+@import "./reset";
+
+.container {
+  display: grid;
+  grid-template-areas: "box box box"
+    "box box box";
+}
+
+.box {
+  font-family: $main-font;
+  font-size: 100px;
+  min-width: 300px;
+  height: $standart-height;
+  color: #fff;
+  text-align: center;
+  line-height: $standart-height;
+}
+
+.box:nth-child(1) {
+  background-color: #fe0202;
+}
+
```

```suggestion
.box:nth-child(1) {
  background-color: #fe0202;
}
```
По BEM з SASS краще задавати nth-child(n) всередині .box:

`
.box {
  &:nth-child(1) {
    background-color: #fe0202;
  }
  
  &:nth-child(2).......
}
`

---

плюс, виправив